scripts/create_corpus.py

```diff
@@ -0,0 +1,112 @@
+import argparse
```

Comment:
I moved this file to `corpus_index` package.
The folder `scripts` is just of Shell scripts and not Python code.
you can delete it from here.